src/systems/camera-system.js

```diff
@@ -334,14 +343,27 @@ export class CameraSystem {
       this.viewingCameraRotator.on = true;
 
       this.userinput = this.userinput || scene.systems.userinput;
-      if (
+      this.interaction = this.interaction || scene.systems.interaction;
+
+      if (this.userinput.get(paths.actions.startInspecting) && this.mode !== CAMERA_MODE_INSPECT) {
+        const hoverEl = this.interaction.state.rightRemote.hovered || this.interaction.state.leftRemote.hovered;
```

Comment:
This will be a bit awkward if ever we have two cursors active at once, but should be fine now since there's only one. I would also double-check that the `this.interaction.state.rightRemote.hovered` state gets cleared when the active remote is switched to the leftRemote. An edge case here that I'm not sure about is if you grab an object with the right remote, then activate your left remote. At that moment I wonder if the right and left remotes are both active and potentially hovering on different elements.

Reply:
Ah, nevermind -- I see now in the `CursorTogglingSystem` that when the left remote becomes active, the hover state of the right remote gets cleared and vice versa.